sklearn/decomposition/kernel_pca.py

```diff
@@ -21,6 +22,13 @@ class KernelPCA(BaseEstimator, TransformerMixin):
     Non-linear dimensionality reduction through the use of kernels (see
     :ref:`metrics`).
 
+    It uses the LAPACK implementation of the full SVD or a randomized truncated
+    SVD by the method of Halko et al. 2009, depending on the shape of the input
```

Comment:
How about "see `eigen_solver`" as the user can explicitly choose any.

Reply:
Agreed. Would this be ok ?

    It uses the LAPACK implementation of the full SVD or a randomized truncated
    SVD by the method of Halko et al. 2009, depending on the shape of the input
    data and the number of components to extract. It can also use the
    scipy.sparse.linalg ARPACK implementation of the truncated SVD, see
    :ref:`eigen_solver`.